### PR TITLE
bump picard version across the board

### PR DESCRIPTION
**Pull request by matthdsm**
I know this isn't the most friendly way to do this, but it's the fastest 🤷🏻‍♂️  
Feel free to close if you don't like it!

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Comment by matthdsm** [comment]
I think I broke conda 😨

File: modules/picard/addorreplacereadgroups/main.nf [hunk 1]
```diff
@@ -43,7 +43,7 @@ process PICARD_ADDORREPLACEREADGROUPS {
         -PL ${PLATFORM} \\
         -PU ${BARCODE} \\
         -SM ${SAMPLE} \\
-        -CREATE_INDEX true
+        --CREATE_INDEX true
```
Comment (maxulysse) [inline modules/picard/addorreplacereadgroups/main.nf:46]: Should we have that in args?
I assume there might be some cases when we actually don't want index

File: modules/picard/createsequencedictionary/main.nf [hunk 2]
```diff
@@ -31,8 +31,8 @@ process PICARD_CREATESEQUENCEDICTIONARY {
         -Xmx${avail_mem}g \\
         CreateSequenceDictionary  \\
         $args \\
-        R=$fasta \\
-        O=${prefix}.dict
+        -R $fasta \\
+        -O ${prefix}.dict
```
Comment (maxulysse) [inline modules/picard/createsequencedictionary/main.nf:35]: Can we use long name parameters everywhere?

File: modules/picard/addorreplacereadgroups/main.nf (hunk 1 above)
Comment (matthdsm) [reply modules/picard/addorreplacereadgroups/main.nf:46]: I have no opinion, I'm just updating everything to the latest notation.

File: modules/picard/createsequencedictionary/main.nf (hunk 2 above)
Comment (matthdsm) [reply modules/picard/createsequencedictionary/main.nf:35]: Fine by me! I'll convert them all

File: modules/picard/addorreplacereadgroups/main.nf (hunk 1 above)
Comment (maxulysse) [reply modules/picard/addorreplacereadgroups/main.nf:46]: I'll create an issue for it then, and we'll figure it out in a separate PR

File: modules/picard/createsequencedictionary/main.nf (hunk 2 above)
Comment (maxulysse) [reply modules/picard/createsequencedictionary/main.nf:35]: I did the same for GATK not so long ago.
Thank you for doing that here

Comment (matthdsm) [reply modules/picard/createsequencedictionary/main.nf:35]: Perhaps it's something that can be added to the guidelines?  Mandatory long params for all tools (where applicable).

File: modules/picard/addorreplacereadgroups/main.nf (hunk 1 above)
Comment (maxulysse) [reply modules/picard/addorreplacereadgroups/main.nf:46]: cf #1588

File: modules/picard/createsequencedictionary/main.nf (hunk 2 above)
Comment (maxulysse) [reply modules/picard/createsequencedictionary/main.nf:35]: Good idea

**Review by maxulysse — APPROVED** [review]

**Review by JoseEspinosa — APPROVED** [review]
Awesome job @matthdsm !
Feel free to merge once tests are passing :rocket:

**Pull request merged**